settings.py

```diff
@@ -50,3 +50,4 @@
 TIME_RECENT_UPDATED = 60
 
 TABLE_GROUP_ALLOWED = {'evpl', 'epl'}
+QUEUE_ID = None
```

Comment:
Could you add a coment above this line summarizing what's it's for, and the supported values? We do have that docs on the openapi.yml but, it's also convenient to have here just so it's immediately clear for whoever is setting the settings.py file (same pattern we've been following for the other exposed vars)

Reply:
Comment added